gymnasium_robotics/envs/multiagent_mujoco/requirements.txt

```diff
@@ -0,0 +1,4 @@
+"gymnasium==0.27.0",
```

Comment:
remove, move to the setup.py as an optional install and add all installs to requirements.txt